src/toil/test/utils/ABC/debugWorkflow.py

```diff
@@ -0,0 +1,177 @@
+from toil.job import Job
+from toil.common import Toil
+import subprocess
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def initialize_jobs(job):
+    job.fileStore.logToMaster('''initialize_jobs''')
+
+def writeA(job, mkFile):
+    job.fileStore.logToMaster('''writeA''')
+
+    # temp folder for the run
+    tempDir = job.fileStore.getLocalTempDir()
+
+    # import files
+    try:
+        mkFile_fs = job.fileStore.readGlobalFile(mkFile[0], userPath=os.path.join(tempDir, mkFile[1]))
+    except:
+        mkFile_fs = os.path.join(tempDir, mkFile[1])
+
+    # make a file (A.txt) and write 'A' into it using 'mkFile.py'
+    content = 'A'
+    cmd = 'python' + ' ' + mkFile_fs + ' ' + 'A.txt' + ' ' + content
+    this_process = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+    this_process.wait()
+
+    # get the output file and return it as a tuple of location + name
+    output_filename = 'A.txt'
+    output_file = job.fileStore.writeGlobalFile(output_filename)
+    A1 = (output_file, output_filename)
+    rvDict = {"A1": A1}
+    return rvDict
+
+def writeB(job, mkFile, B_file):
+    job.fileStore.logToMaster('''writeB''')
+
+    # temp folder for the run
+    tempDir = job.fileStore.getLocalTempDir()
+
+    # import files
+    try:
+        mkFile_fs = job.fileStore.readGlobalFile(mkFile[0], userPath=os.path.join(tempDir, mkFile[1]))
+    except:
```

Comment:
I've thought about this, and it won't conceal any errors.  Basically, it's setting a file path.  Either one given to it by importing a file into the jobstore, or if that fails (because that file already exists in the jobstore), create the path as a string.  If that file doesn't exist or isn't available, you will definitely know, and it will raise an error later when it tries to use that path.